meerkat/interactive/app/src/lib/component/abstract.py

```diff
@@ -304,15 +301,16 @@ def _frontend(value):
             base_types=(Store),
         )
         return ComponentFrontend(
```

Comment:
@seyuboglu @ad12 
Take a look at this change: I think we can just use the frontend alias here (i.e. if a component `X` subclassed `div` we currently send `X` over to the frontend instead of `div`). Just want to make sure I'm not missing something here.

Reply:
yea that should work just fine - perhaps adding a test that the frontend_alias maps to the appropriate component would be good